modules/API/API.js

```diff
@@ -273,32 +232,17 @@ class API {
      * @returns {void}
      */
     notifyReadyToClose() {
-        triggerEvent('video-ready-to-close', {});
+        this._sendEvent('video-ready-to-close', {});
     }
 
     /**
-     * Sends remote control event.
-     *
-     * @param {RemoteControlEvent} event - The remote control event.
-     * @returns {void}
-     */
-    sendRemoteControlEvent(event) {
-        sendMessage({
-            method: 'remote-control-event',
-            params: event
-        });
-    }
-
-    /**
-     * Removes the listeners.
+     * Disposes the allocated resources.
      *
      * @returns {void}
      */
     dispose() {
-        if (postis) {
-            postis.destroy();
-            postis = undefined;
-
+        if (this.enabled) {
+            this.enabled = false;
```

Comment:
This is implemented in https://github.com/jitsi/jitsi-meet/pull/1539 . But we are doing dispose at the end which should remove all listeners.